Description: link to gl as libglu-dev doesn't link automatically to libgl now
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-04-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -22,8 +22,8 @@
 
 # Detect OS --------------------------------------------------------------------
 ifneq ($(strip $(shell $(CC) -v 2>&1 |grep "linux")),)
-	LDFLAGS += $(shell $(PKG_CONFIG) --libs sdl glu zlib libpng) -lfmod-3.75 $(GUILIB)
-	CFLAGS += $(shell $(PKG_CONFIG) --cflags sdl glu zlib libpng) $(GUIOPT)
+	LDFLAGS += $(shell $(PKG_CONFIG) --libs sdl glu gl zlib libpng) -lfmod-3.75 $(GUILIB)
+	CFLAGS += $(shell $(PKG_CONFIG) --cflags sdl glu gl zlib libpng) $(GUIOPT)
 endif
 
 ifneq ($(strip $(shell $(CC) -v 2>&1 |grep "Apple")),)
